templates/jfrog-xray-ec2-instance.template.yaml

```diff
@@ -61,93 +69,76 @@ Parameters:
     ca-central-1:
-      "330": ami-0148cebea7bea4aaf # TODO: Get correct ami - provided by market place tem
-      "361": ami-0148cebea7bea4aaf # TODO: Get correct ami - provided by market place tem
+      CentOS7HVM: "ami-07423fb63ea0a0930"
     eu-central-1:
-      "330": ami-07961f7c210143a42 # TODO: Get correct ami - provided by market place tem
-      "361": ami-07961f7c210143a42 # TODO: Get correct ami - provided by market place tem
+      CentOS7HVM: "ami-0cfbf4f6db41068ac"
     eu-west-1:
-      "330": ami-0171b8d46941b4ca1 # TODO: Get correct ami - provided by market place tem
-      "361": ami-0171b8d46941b4ca1 # TODO: Get correct ami - provided by market place tem
+      CentOS7HVM: "ami-08935252a36e25f85"
     sa-east-1:
-      "330": ami-0596f196b273bb8a6 # TODO: Get correct ami - provided by market place tem
-      "361": ami-0596f196b273bb8a6 # TODO: Get correct ami - provided by market place tem
+      CentOS7HVM: "ami-05145e0b28ad8e0b2"
     us-east-1:
-      "330": ami-0d4d4252cdc2b6f11 # TODO: Get correct ami - provided by market place tem
-      "361": ami-086fcbf4aa2bd203f # TODO: Get correct ami - provided by market place tem
-      "386": ami-0becff949aa530956 # partnership account + seller account
-      "3103": ami-0e19b1335bc3654c3	 # seller account (shared with partnership account)
-      "3112": ami-0819678d7216af530 # seller account (shared with partnership account)
-      "3120": ami-0a6172e50d36025e5 # seller account (shared with partnership account)
+      CentOS7HVM: "ami-0affd4508a5d2481b"
     us-east-2:
-      "330": ami-00a5fcde44618d39b # TODO: Get correct ami - using ami generated by myself - provided by market place tem
-      "361": ami-005b2ceceac6999ff # TODO: Get correct ami - using ami generated by myself - provided by market place tem
-      "3112": ami-0bd793595d742f794 # seller account (shared with partnership account)
-      "3120": ami-0584c00709af28577 # seller account (shared with partnership account)
+      CentOS7HVM: "ami-01e36b7901e884a10"
     us-west-1:
-      "330": ami-068cd684b4d3a3a86 # TODO: Get correct ami - provided by market place tem
-      "361": ami-068cd684b4d3a3a86 # TODO: Get correct ami - provided by market place tem
-      "3112": ami-012cc5d182bd3bd2b # seller account (shared with partnership account)
-      "3120": ami-02d0a6ecbadc0a0bd # seller account (shared with partnership account)
+      CentOS7HVM: "ami-098f55b4287a885ba"
     us-west-2:
-      "330": ami-03d60da4c8a146a55 # TODO: Get correct ami - provided by market place tem
-      "361": ami-03d60da4c8a146a55 # TODO: Get correct ami - provided by market place tem
-      "386": ami-07af1682f09ef4a20 # partnership account + seller account
-      "3112": ami-0b2006b832e129368 # seller account (shared with partnership account)
-      "3120": ami-0167c1be5fc3a84cc # seller account (shared with partnership account)
+      CentOS7HVM: "ami-0bc06212a56393ee1"
     us-gov-east-1:
```

Comment:
Since for Gov clouds we need Marketplace templates, these templates can not be used in gov cloud, so we can remove gov clouds from here and also from artifactory-instance templates